packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -853,6 +882,24 @@ export class Project extends ProjectBase {
     }
   }
 
+  public enableBatchBuilds(): IEnableBatchBuildsReturn | undefined {
+    if (!this._batchServiceRole) {
+      this._batchServiceRole = new iam.Role(this, 'BatchServiceRole', {
+        roleName: PhysicalName.GENERATE_IF_NEEDED,
```

Comment:
Curious, why this here? Do you expect this Role to be used in a cross-env fashion?

Reply:
Don't think so. I just copied it from somewhere else in this file 😆

Will remove